api/types/crds/skupper_site_crd.yaml

```diff
@@ -17,27 +17,47 @@ spec:
               properties:
                 routerMode:
                   type: string
+                  description: |
+                    Defines the mode of the router for the site.
+                    - **interior**: Full linking capability across the network.
+                    - **edge**: Limited to linking to an interior site, acts as a leaf node.
                 serviceAccount:
                   type: string
+                  description: |
+                    The site uses the specified service account to interact with Kubernetes resources.
```

Comment:
Maybe 'If specified' at the start, and then state that if not specified a serviceaccount will be created by the controller.